src/manifest/dbtProject.ts

```diff
@@ -188,6 +190,53 @@ export class DBTProject implements Disposable {
     return this.dbtProjectIntegration.getMacroPaths();
   }
 
+  getManifestPath() {
+    const targetPath = this.getTargetPath();
+    if (!targetPath) {
+      return;
+    }
+    return path.join(targetPath, "manifest.json");
+  }
+
+  getCatalogPath() {
+    const targetPath = this.getTargetPath();
+    if (!targetPath) {
+      return;
```

Comment:
should this be error?

Reply:
have added check at usage of the function